cartography/intel/sentinelone/cve.py

```diff
@@ -0,0 +1,118 @@
+import logging
+from typing import Any
+
+import neo4j
+
+from cartography.client.core.tx import load
+from cartography.graph.job import GraphJob
+from cartography.intel.sentinelone.api import get_paginated_results
+from cartography.intel.sentinelone.utils import get_application_version_id
+from cartography.models.sentinelone.cve import S1CVESchema
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get(api_url: str, api_token: str, account_id: str) -> list[dict[str, Any]]:
+    """
+    Fetch CVE data from SentinelOne API
+    Should be simple and raise exceptions on failure
+    """
+    logger.info("Retrieving SentinelOne CVE data")
+    cves = get_paginated_results(
+        api_url=api_url,
+        endpoint="/web/api/v2.1/application-management/risks",
+        api_token=api_token,
+        params={
+            "limit": 1000,
+            "accountIds": account_id,
+        },
+    )
+
+    logger.info("Retrieved %d CVEs from SentinelOne", len(cves))
+    return cves
+
+
+def transform(cves_list: list[dict[str, Any]]) -> list[dict[str, Any]]:
+    """
+    Transform SentinelOne CVE data for loading into Neo4j
+    """
+    transformed_cves = []
+    for cve in cves_list:
+        app_version_id = get_application_version_id(
+            cve.get("applicationName", "unknown"),
+            cve.get("applicationVendor", "unknown"),
+            cve.get("applicationVersion", "unknown"),
+        )
+        transformed_cve = {
+            # Required fields - let them fail if missing
+            "id": cve["id"],
+            "cve_id": cve["cveId"],
+            # Optional fields - use .get() with None default
+            "application_version_id": app_version_id,
```

Comment:
Is `app_version_id` optional as well hence why the `"unknown"` values are accepted?